documentation/decisions/adr/0003-api-language.md

```diff
@@ -0,0 +1,95 @@
+# API Language
+
+- **Status:** Accepted 
+- **Last Modified:** 2023-06-30
+- **Related Issue:** [#27](https://github.com/HHS/grants-api/issues/27) 
+- **Deciders:** Lucas Brown, Aaron Couch, Sammy Steiner, Daphne Gold, Billy Daly
+
+## Context and Problem Statement
+
+This ADR is to decide what programming language the API server for beta.grants.gov will use. This API server will initially be responsible for responding to queries to search a replicated database, but will grow to encompass all the backend responsibilities of grants.gov
+
+In addition to the Nava and HHS teams ability to develop and maintain an API in the chosen language, it is also important that this language lends itself to open source engagement.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- Nava and HHS familiarity with language and common libraries
+- Performance of language and associated frameworks at scale
+- Support of Open Source community
+- Cost of necessary licenses, support, etc.
+
+## Options Considered
+
+- Python
+- Javascript and Node
+- Java
+
+
+## Decision Outcome <!-- REQUIRED -->
+Chosen option: Python, because it is free and open source and designed to be easy for anyone to learn quickly and contribute. Additionally, while all the languages can support the technical needs, python in particular is optimized to handle the large and complex grants.gov dataset, and makes other parts of the project simpler, including ETL and data analysis for the analytics endpoints. 
```

Comment:
```suggestion
Chosen option: Python, because it is free and open source and designed to be easy for anyone to learn quickly and contribute. Additionally, while all the languages can support the technical needs, Python in particular is optimized to handle the large and complex grants.gov dataset, and makes other parts of the project simpler, including ETL and data analysis for the analytics endpoints. 
```